packages/zosfiles/__tests__/api/methods/create/Create.unit.test.ts

```diff
@@ -139,6 +140,24 @@ describe("Create data set", () => {
             );
         });
 
+        it("should be able to allocate like from a sequential data set", async () => {
```

Comment:
Please add an API system test in `packages/zosfiles/__tests__/__system__/api/methods/create` modeled after this test.

Reply:
Please add an API system test. I think there was one previously, but it seems to have gone missing.

Reply:
Added